src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -60,7 +60,11 @@ class PetController (private val petService: PetService) {
 
     @POST
```

Comment:
This is update - not create - so should be @PUT and not @POST

Reply:
fixed